src/core/AudioEngine/AudioEngine.cpp

```diff
@@ -682,7 +659,7 @@ void AudioEngine::setAudioDriver( AudioOutput* pAudioDriver ) {
 	this->unlock();
 	
 	if ( m_pAudioDriver != nullptr &&
-		 m_pAudioDriver->class_name() != DiskWriterDriver::_class_name() ) {
+		 dynamic_cast<DiskWriterDriver*>(m_pAudioDriver) == nullptr ) {
```

Comment:
Maybe it would be better to make the DDWD trivially `connect()` to avoid this? Or maybe having an `isDisconnectedDriver()` method that answers `true` for the DDWD (and FakeDriver?)